pandas/_typing.py

```diff
@@ -382,11 +381,9 @@ def closed(self) -> bool:
 ]
 
 # internals
-Manager = Union[
-    "ArrayManager", "SingleArrayManager", "BlockManager", "SingleBlockManager"
-]
-SingleManager = Union["SingleArrayManager", "SingleBlockManager"]
-Manager2D = Union["ArrayManager", "BlockManager"]
+Manager = Union["BlockManager", "SingleBlockManager"]
+SingleManager: TypeAlias = "SingleBlockManager"
+Manager2D: TypeAlias = "BlockManager"
```

Comment:
Not sure if these are still useful - can remove

Reply:
Only useful if they are used elsewhere in the code

Reply:
They are used - but I'm not sure that makes them useful. We can replace any current usage of SingleManager with SingleBlockManager.

I tracked it down, this was added in https://github.com/pandas-dev/pandas/pull/40152/, so I think it makes sense to remove.